tests/subsys/bootloader/provision/prj.conf

```diff
@@ -12,6 +12,5 @@ CONFIG_SECURE_BOOT=y
 CONFIG_FW_INFO=y
 CONFIG_REBOOT=y
 CONFIG_FW_INFO_FIRMWARE_VERSION=10
-CONFIG_SECURE_BOOT_PURSE=y
 CONFIG_SECURE_BOOT_CRYPTO=y
-CONFIG_SB_NUM_MONOTONIC_COUNTERS=3
+CONFIG_SB_NUM_MONOTONIC_COUNTERS=4
```

Comment:
I think you want the entire file added here, not just the move/change.

Reply:
"here" as in this commit

Reply:
Fixed